packages/next/apollo.jsx

```diff
@@ -43,20 +46,49 @@ let createApolloClient = (initialState, ctx, apolloCacheConfig, apolloClientConf
         }
     }
 
+    const hasFiles = (data) => {
+        let i = 0
+        let found
+
+        const keys = Object.keys(data)
+        try {
```

Comment:
Looks cheeky

Reply:
Thats was for debug. Removed try catch block